packages/patternfly-4/react-integration/cypress/integration/applicationlauncher.spec.ts

```diff
@@ -0,0 +1,24 @@
+describe('Application Launcher Demo Test', () => {
+  it('Navigate to demo section', () => {
+    cy.visit('http://localhost:3000/');
+    cy.get('#application-launcher-demo-nav-item-link').click();
+    cy.url().should('eq', 'http://localhost:3000/application-launcher-demo-nav-link');
+  });
+ 
+  it('Verify application launcher button', () => {
+    cy.get('button').should('have.class', 'pf-c-button pf-m-plain');
+  });
+
+  it('Verify app launcher menu toggle', () => {
+    cy.get('#pf-random-id-0').click();
+    cy.get('ul').should('have.class', 'pf-c-app-launcher__menu');
+    cy.get('#pf-random-id-0').click(); 
+    cy.get('.pf-c-app-launcher__menu').should('not.exist'); 
+  });
+
+  it('Verify onSelect', () => {
+    cy.get('#pf-random-id-0').click(); 
+    cy.get('.pf-c-app-launcher__menu-item').first().click(); 
+    cy.log('This works');
```

Comment:
This is not verifying anything?

Reply:
I had a lot of trouble testing a console log. Any thoughts on other ways to do this? I've also tried: 
```
it('Verify onSelect', () => {
    cy.window().then(win => {
      const spy = cy.spy(win.console, "log");
      cy.get('#pf-random-id-0').click(); 
      cy.get('.pf-c-app-launcher__menu-item').first().click();
      win.console.log('hi'); 
      expect(spy).to.be.called.callCount(2);
    });
  });
```

Reply:
Can you have the demo do something like navigate to another page or print text and verify that?